coalib/parsing/LineParser.py

```diff
@@ -56,6 +58,17 @@ def parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+        if line.startswith('#'):
+            if(line[1] != ' '):
+                logging.warning('Your line has no whitespace after #')
+
+        if (line != '' and line.find('#') != -1):
+            pos = [i for i, x in enumerate(line) if x == '#']
```

Comment:
...in fact, if we have the `comment_start` saved, we could just use it for the above check as well and do

```
if comment_start = 0 and line[1] != ' ':
```

there.